index.bs

```diff
@@ -10,6 +10,19 @@ Editor: Thomas Steiner 44965, Google Inc., https://google.com/
 Editor: Marijn Kruisselbrink 72440, Google Inc., https://google.com/
 Group: dap
 Status Text:
+  <strong>This specification presents a [[#use-cases-categorization]] informed by feedback
+  and requests from the wider community, end users and web developers. These use cases
+  welcome further review from the wider web community to ensure the documented use case set
+  satisfies the needs of all stakeholders and provides appropriate abstractions for these capabilities.
+  Review and feedback is sought after in particular for the new [[#use-cases-background-continuous]]
+  and [[#use-cases-background-geofence]] use case categories. All feedback, including
+  contributions for new informative use cases and other considerations,
+  is welcome via <a href="https://github.com/w3c/geolocation-sensor/issues">GitHub issues</a>.
```

Comment:
```suggestion
  is welcome via <a href="[REPOSITORYURL]/issues">GitHub issues</a>.
```

Reply:
Thanks @reillyeon. Addressed your suggestion in an update, also updated the commit message.

Could you open a new issue linking to the [proposal from 2014](https://lists.w3.org/Archives/Public/public-geolocation//2014Sep/0012.html) you mentioned?

I'd like to link to that issue from this status text as to raise awareness of options that have been considered and to invite feedback and comments from the web community on that proposal.

Reply:
It looks like spec-prod validator does not like [REPOSITORYURL] text macro inside href, CI error:

>error: Bad value “[REPOSITORYURL]/issues” for attribute “href” on element “a”: Illegal character in path segment: “[” is not allowed.

Local build was fine. May need to revert back to the plain old URL unless some Bikeshed expert tells me otherwise.

Reply:
We don't define repository in macro, so add repository line (with `Repository: w3c/geolocation-sensor`)

Reply:
Filed https://github.com/w3c/geolocation-sensor/issues/62.

Reply:
PR updated. The rendered version will have a very visible issue block on top of the API section pointing to that issue. The status text links to the API section too setting expectations with regard to changes expected.

Also fixed the text macro, thanks @himorin for filing an issue for that.

Reply:
This change is fine but please remove the "Fix #60" comment in the description as this does not resolve the issue raised there. To be clear, I think there are two paths forward for resolving that issue. Either,

1. Mark this specification as Obsolete and start a new specification tailored to the background/geofencing use cases.
2. Get consensus from working group participants that the API presented in the current draft is something that should be pursued as a replacement for the existing Geolocation API.

I recommend (1). I don't foresee (2) being possible because there has been no interest for many years in implementing this API.

Reply:
@reillyeon thanks for the suggestions. I removed the left-over "Fix" from the comment. PTAL and merge at will.

Some considerations:

- Background/geofencing use cases and path forward will be an excellent TPAC discussion.
- If the WG finds out relevant foreground use cases are addressed by the old Geolocation API, this new API could be rescoped to address background/geofencing use cases.
- For background/geofencing API shape considerations, @mkruisselbrink's proposals referenced in #62 are (still) good input and this PR makes those proposals prominent and notes the API is expected to change. (Good proposals age like fine wine! 🍷)

A process-related clarification for option (1):

Given this spec is a Working Draft, and given (1) proposes a narrower scope, the WG only needs to decide to publish an updated Working Draft with foreground functionality removed without any heavy-handed mechanisms beyond that.

Reply:
Good point, we can just modify the Working Draft.